srcpkgs/nextdns/files/nextdnsd/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec nextdns run -config-file /etc/nextdns/nextdns.conf 2>&1
```

Comment:
You can add a comment above this line, like `# See https://example.com/doc for example configuration`

Reply:
There isn't a clear documentation about config file, the only helpful link is this  https://github.com/nextdns/nextdns/wiki/Configuration-File-Format but it doesn't explain that you have to use your config id etc...
So I think that the best thing to do is to provide an example config-file in doc dir, so users are free to use it or do more research them self.

Reply:
@gspe: The best thing to do in this case is to ask upstream to improve their documentation, in the form of a well-commented example config file[1]. Upstream knows the software best, and can make sure such a file is properly maintained and contains the correct information. We shouldn't be trying to take on that task.

[1] Cf. e.g. the extensively-commented sample `wpa_supplicant.conf` file provided by upstream.